internal/schemautil/schemautil.go

```diff
@@ -336,3 +338,30 @@ func unmarshalUserConfig(src interface{}) ([]map[string]interface{}, error) {
 
 	return []map[string]interface{}{config}, nil
 }
+
+// importingResourceID Stores `terraform import` resource id
+// The terraform import command can only import one resource at a time
+// https://developer.hashicorp.com/terraform/cli/import/usage
+var (
+	importingResourceID         string
+	importingResourceIDSentinel int32
+)
+
+// ImportStatePassthroughContext same as schema.ImportStatePassthroughContext
+// but stores importing resource id to now if Read() handler has been called on `terraform import` command
+func ImportStatePassthroughContext(ctx context.Context, d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
```

Comment:
It looks to me like an overkill, I think we can simplify this to something like: 
```
StateContext: func(ctx context.Context, d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
	if err := d.Set("compatibility_level", "importing"); err != nil {
		return nil, aiven.ErrInvalidHost
	}
	return schema.ImportStatePassthroughContext(ctx, d, m)
},
``` 

WDYT?